### ISSUE TITLE
Can't invoke FFmpeg methods(FFmpeg.getInstance() and new ExecuteBinaryResponseHandler())

### ISSUE THREAD

**Issue by rennovale** [bravobit/FFmpeg-Android#131]
I'm trying to use FFmpeg if Android Studio(java) with a device J415G Android Pie SDK 28(if i'm not mistaken)

The problem is that i can use FFmpeg calling the command without problems:

> FFmpeg.execute("--here goes the commands");

BUT, i can't find/call some important methods(they weren't defined):
>FFmpeg.getInstance() 
>ExecuteBinaryResponseHandler()

I found only 3 FFmpeg.execute() definitions:
1) public static int execute(final String command) 
2) public static int execute(final String command, final String delimiter) 
3) public static int execute(final String[] arguments)

None of those arguments expected a new ExecuteBinaryResponseHandler()

In other words i can't check if executed commands were successful, can't check if FFmpeg is supported, and so on..
> FFmpeg.getInstance(this).isSupported()
![Capturar](https://user-images.githubusercontent.com/19192837/63211521-bdb40b80-c0ce-11e9-9b39-2e20aa9cf055.PNG)

What i did to "install" FFmpeg in Android Studio was add the following line in gradle file. As expected method FFmpeg.execute works, but can't use most of others FFmpeg methods...

>dependencies {
    implementation 'nl.bravobit:android-ffmpeg:1.1.7'
}

Is this a issue or am i doing something wrong? 
Sorry for any trouble i'm very newbie in Android development...
And thanks for help

**Comment by rennovale** [comment]
Sorry i imported two libraries by mistake and was trying to use bravobit commands with the another ffmpeg library...
